wavenet_params.json

```diff
@@ -1,11 +1,15 @@
 {
     "filter_width": 2,
     "sample_rate": 16000,
-    "dilations": [1, 2, 4, 8, 16, 32, 64, 128, 256,
-                  1, 2, 4, 8, 16, 32, 64, 128, 256],
+    "dilations": [1, 2, 4, 8, 16, 32, 64, 128, 256, 512,
+                  1, 2, 4, 8, 16, 32, 64, 128, 256, 512,
+                  1, 2, 4, 8, 16, 32, 64, 128, 256, 512,
+                  1, 2, 4, 8, 16, 32, 64, 128, 256, 512],
     "residual_channels": 32,
-    "dilation_channels":16,
+    "dilation_channels":32,
     "quantization_channels": 256,
-    "skip_channels": 256,
-    "use_biases": false
+    "skip_channels": 512,
+    "use_biases": false,
+    "scalar_input": true,
```

Comment:
Default to "scalar_input": false, until we've proven that it works as well or better?